document/document.js

```diff
@@ -25,7 +25,7 @@ var globals = require('can-globals/can-globals-instance');
  * @return {Object} The window object for this JavaScript environment.
  */
 globals.define('document', function(){
-	return globals.getKeyValue('global').document;
+	return globals.getKeyValue('document') || globals.getKeyValue('global').document;
```

Comment:
What’s the purpose of this change?

Reply:
I don't understand this either. This is the default implementation. When someone does `globals.getKeyValue('document')` this is what it will do by default.  Once you've set another document this code won't run any more.

If you could provide a failing test case it would help with understanding the problem better.